tests/common/PlatformInfo.cs

```diff
@@ -50,6 +52,20 @@ static PlatformInfo GetHostPlatformInfo ()
 
 			var platformInfo = new PlatformInfo ();
 
+#if NET
+			if (name.StartsWith ("maccatalyst", StringComparison.Ordinal))
+				platformInfo.Name = ApplePlatform.MacCatalyst;
+			else if (name.StartsWith ("mac", StringComparison.Ordinal))
+				platformInfo.Name = ApplePlatform.MacOSX;
+			else if (name.StartsWith ("ios", StringComparison.Ordinal) || name.StartsWith ("iphoneos", StringComparison.Ordinal) || name.StartsWith ("ipados", StringComparison.Ordinal))
+				platformInfo.Name = ApplePlatform.iOS;
+			else if (name.StartsWith ("tvos", StringComparison.Ordinal))
+				platformInfo.Name = ApplePlatform.TVOS;
+			else if (name.StartsWith ("watchos", StringComparison.Ordinal))
+				platformInfo.Name = ApplePlatform.WatchOS;
+			else
+				throw new FormatException ($"Unknown product name: {name}");
```

Comment:
does not matter, but you could have used a switch + pattern matching:
```csharp
switch(n)
{
    case string name when name.StartsWith("tvos", StringComparison.Ordinal)):
       return "123";
    case string name when name.StartsWith("watchos", StringComparison.Ordinal):
       return "123";
    default:
       return "ABCXYZ123";
}
```

Reply:
heh, might've been a way to reduce code duplication by having something like:

```csharp
#if NET
using PlatformName = ApplePlatform;
#endif
```